lib/index.js

```diff
@@ -20,7 +20,8 @@ var versions = {
   '2.x.x': 'fs-2.0.0.js',
   '3.x.x': 'fs-3.0.0.js',
   '4.x.x': 'fs-4.0.0.js',
-  '5.x.x': 'fs-5.0.0.js'
+  '5.x.x': 'fs-5.0.0.js',
+  '6.x.x': 'fs-5.0.0.js'
```

Comment:
The file you committed is `fs-6.0.0`, but here you have `fs-5.0.0.js`. Presumably a typo?